workspace.go

```diff
@@ -487,6 +508,15 @@ type WorkspaceUpdateOptions struct {
 	// repository.
 	WorkingDirectory *string `jsonapi:"attr,working-directory,omitempty"`
 
+	// Struct containing boolean values that indicate whether the resolved value of
+	// a setting should be decided by something other than the workspace (i.e. a related
+	// organization or project).
+	//
+	// For example: When setting the execution-mode of a workspace to  "default", the
+	// `setting-overwrites.execution-mode` field should be set to true, because the execution-mode
```

Comment:
Similar to previous comment, this is backwards. False means inheriting, true means overwriting

Reply:
Same response as last one.